frontend/src/libraries/explorer-wamp/nodes.ts

```diff
@@ -30,7 +30,7 @@ export default class NodesApi extends ExplorerApi {
   ) {
     let whereClause = `WHERE last_seen > (strftime('%s','now') - 60) * 1000 `;
     if (endTimestamp) {
-      whereClause += ` AND last_seen < :endTimestamp`;
+      whereClause += ` AND node_id < :endTimestamp`;
```

Comment:
`node_id < :endTimestamp` is crazy :cry:

Reply:
I will change `endTimestamp` to `paginationIndexer`

Reply:
change it in another PR since this one is for node list issue.